apps/labs/posts/sympy_documentation.md

```diff
@@ -0,0 +1,356 @@
+
+Based on the results of the survey, we decided to use the Furo theme. The Furo
+theme was ranked the highest by survey respondents. In particular, they liked
+the improved sidebar navigation, the dark mode, and mobile support.
+Additionally, Furo has good accessibility and the CSS is easy to customize.
+
+The result is a documentation site that has navigable sidebars. We spent
+considerable time retheming the default Furo colors to match the traditional
+SymPy green theme. This included adding a dark mode set of colors (this can be
+accessed by clicking the circle icon at the top of a docs page, or by setting
+your device to use dark mode). We took care to make sure all color
+combinations used throughout the documentation were at least WCAG level
+AA color contrast so that text can be perceived by a wide audience
+of readers, including many with low vision. This included
+modifying the [Pygments syntax highlighting
+styles](https://github.com/sympy/sympy/blob/master/doc/src/_pygments/styles.py)
+to have better color contrast.
+
+# Improved Organization - Diataxis
+
+![Main page of the SymPy documentation from SymPy 1.12 (October 2023)](/posts/sympy-documentation/sympy-112-docs-main-page.png)
+![Main page of the SymPy documentation from SymPy 1.7 (May 2021)](/posts/sympy-documentation/sympy-17-docs-main-page.png)
+
+Docs for the main SymPy documentation landing page
+[now](https://docs.sympy.org/latest/index.html) vs. in [early 2021](https://web.archive.org/web/20210225051926/https://docs.sympy.org/latest/index.html)
+(courtesy of [Wayback Machine](https://archive.org/web/)).
+
+A related project was reorganizing the top-level organization of the
+documentation. As you can see from the screenshot above from early 2021, the
+SymPy documentation main page used to just consist of a long list of every
+page in the documentation. The new Furo theme makes this list unnecessary, but
+it also gave us an opportunity to explore how these pages could be explored
+in a more logical way.
+
+We decided to adopt the [Diátaxis](https://diataxis.fr/) framework for documentation
+organization. Diátaxis splits documentation pages into one of
+four categories, depending on whether the reader is interested in practical or
+theoretical knowledge, and on whether they have study or work oriented goals.
+As can be seen from the current documentation main page, the docs are now
+organized into four categories: tutorials, how-to guides, explanations, and
+API reference. We additionally added "installation" and "contributing" (see
+below) as separate top-level categories. Installation is important enough to
+warrant calling out documentation for it separately. Contribution documentation
+is separate because it serves a separate audience, people who want to
+contribute to SymPy, rather than people who are interested in using it.
+
+<!-- TODO: Add Diátaxis framework diagram here -->
```

Comment:
Same, MDX comment:

```suggestion
{/* TODO: Add Diátaxis framework diagram here */}
```

Reply:
Looks like I can use the image from https://diataxis.fr/ is [CC-BY-SA](https://github.com/evildmp/diataxis-documentation-framework/blob/main/LICENSE.rst). Is it OK to use that? Otherwise, let's just not worry about including this diagram.